Update article tableaux

pya35 opened the pull request:
FR : https://deploy-preview-784--a11y-guidelines.netlify.app/fr/articles/tableau-accessible/

pya35 committed:
add updateDate

pya35 committed:
first update

pya35 committed:
typo

pya35 committed:
Merge branch 'master' into update-article-tableaux

pya35 committed:
typo

pya35 committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -105,29 +113,29 @@ Le fait de créer des tableaux accessibles permettra une lecture cohérente de c
-<li><kbd>Ctrl + alt + flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl + alt + flèche droite</kbd> se déplace sur la colonne de droite.</li>
-<li><kbd>Ctrl + alt + flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl + alt + flèche haut</kbd> pour passer à la ligne précédente.</li>
+<li><kbd>Ctrl + Alt + flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl + Alt + flèche droite</kbd> se déplace sur la colonne de droite.</li>
+<li><kbd>Ctrl + Alt + flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl + Alt + flèche haut</kbd> pour passer à la ligne précédente.</li>
```
```suggestion
<li><kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche haut</kbd> pour passer à la ligne précédente.</li>
```

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -193,15 +201,15 @@ Le premier exemple est un tableau avec seulement des en-têtes sur les colonnes,
+Ce tableau nécessite deux en-têtes, un en-tête de colonne pour les jours de la semaine et un en-tête de ligne pour le créneau horaire.
 
 <table class="table">
  <caption class="h4">Ouverture du magasin de jouets</caption>
```
The word `jouets` is written without an `s` in the code example just below the `table` (line 259).

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -11,43 +12,52 @@ tags:
-Il doit être associé au tableau grâce à la balise <code>caption</code> et doit être le premier élément après la balise d'ouverture <code>table</code>. On peut aussi utiliser un titre <code>h1,h2...</code> placé dans le code juste avant le tableau comme autre moyen d'associer un titre.
-
-#### Exemple de <code>caption</code>
+Il est important de définir un titre à votre tableau. À la fois concis et pertinent, ce texte doit permettre de comprendre la nature ainsi que le type de données qu'il contient.
```
```suggestion
Il est important de donner un titre à votre tableau. À la fois concis et pertinent, ce texte doit permettre de comprendre la nature ainsi que le type de données qu'il contient.
```

pya35 committed:
Fix scope attribute for column headers in table example

pya35 committed:
Apply suggestions from code review

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Fix caption text in accessible table example

pya35 committed:
Correct caption text in accessible table

pya35 committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -11,43 +12,52 @@ tags:
 </code></pre>
 
-#### Exemple de titre HTML
+Une autre façon de donner un titre est l'utilisation des attributs <code>aria-labelledby</code> ou <code>aria-label</code> sur la balise <code>table</code>.
```
A title for the table but not a `title` attribute.

pya35 committed:
Enhance explanation of table title attributes

Clarified the use of aria-labelledby and aria-label attributes for table titles.

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -193,15 +201,15 @@ Le premier exemple est un tableau avec seulement des en-têtes sur les colonnes,
+De cette manière, il est possible de naviguer facilement au sein du tableau à l'aide d'un lecteur d'écran, tout changement de cellule (d'une colonne, ou d'une ligne à l'autre) étant restitué avec l'en-tête associé.
 
-Par Exemple, si on est positionné sur la colonne Prénom, et qu'on utilise le raccourci <kbd>Ctrl+alt+fleche droite</kbd> pour passer à la colonne Nom, NVDA vocalise "Nom Colonne 2 + texte de la colonne".
+Par exemple, si on est positionné sur la colonne "Prénom" et qu'on utilise le raccourci <kbd>Ctrl+Alt+flèche droite</kbd> pour passer à la colonne "Nom", NVDA vocalise _"Nom Colonne 2 + texte de la colonne"_.
```
```suggestion
Par exemple, si on est positionné sur la colonne "Prénom" et qu'on utilise le raccourci <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche droite</kbd> pour passer à la colonne "Nom", NVDA vocalise _"Nom Colonne 2 + texte de la colonne"_.
```

MewenLeHo reviewed (commented):
Ok for the french version.

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -105,35 +113,35 @@ Le fait de créer des tableaux accessibles permettra une lecture cohérente de c
+<li><kbd>Ctrl + Alt + flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl + Alt + flèche droite</kbd> se déplace sur la colonne de droite.</li>
+<li><kbd>Ctrl + Alt + flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl + Alt + flèche haut</kbd> pour passer à la ligne précédente.</li>
+<li><kbd>Insert + Maj + flèche haut</kbd> permet de lire toute la ligne.</li>
+<li><kbd>Insert + Maj + num pad 5</kbd> permet de lire toute la colonne.</li>
```
```suggestion
<li><kbd>Insert</kbd> + <kbd>Ctrl</kbd> + <kbd>T</kbd> permet de lister tous les tableaux.</li>
<li><kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche droite</kbd> se déplace sur la colonne de droite.</li>
<li><kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche haut</kbd> pour passer à la ligne précédente.</li>
<li><kbd>Insert</kbd> + <kbd>Maj</kbd> + <kbd>flèche haut</kbd> permet de lire toute la ligne.</li>
<li><kbd>Insert</kbd> + <kbd>Maj</kbd> + <kbd>num pad 5</kbd> permet de lire toute la colonne.</li>
```

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -105,35 +113,35 @@ Le fait de créer des tableaux accessibles permettra une lecture cohérente de c
 <ul>
-<li><kbd>Ctrl + alt + flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl + alt + flèche droite</kbd> se déplace sur la colonne de droite.</li>
-<li><kbd>Ctrl + alt + flèche bas</kbd> permet de passer à la ligne suivante en restant sur la même colonne, <kbd>Ctrl + alt + flèche haut</kbd> pour passer à la ligne précédente.</li>
+<li><kbd>Ctrl<kbd> + <kbd>Alt</kbd> + <kbd>flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche droite</kbd> se déplace sur la colonne de droite.</li>
```
```suggestion
<li><kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche gauche</kbd> permet de se déplacer sur la colonne de gauche tout en gardant la même ligne, <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>flèche droite</kbd> se déplace sur la colonne de droite.</li>
```

MewenLeHo committed:
Update src/fr/articles/tableau-accessible.md

pya35 committed:
Update src/fr/articles/tableau-accessible.md

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Update src/fr/articles/tableau-accessible.md

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Merge branch 'master' into update-article-tableaux

pya35 committed:
update after review

pya35 committed:
Merge branch 'update-article-tableaux' of https://github.com/Orange-OpenSource/a11y-guidelines into update-article-tableaux

pya35 committed:
update english

pya35 committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -11,43 +12,55 @@ tags:
 ## Présentation générale
 
-Un tableau est un agencement d'informations en lignes et en colonnes contenant des cellules facilitant la comparaison et la mise en avant des informations. Ils permettent de présenter des informations tabulaires dans une grille en deux dimensions, ces données sont plus faciles à lire sous forme de tableau. 
+Un tableau est un agencement d'informations en lignes et en colonnes contenant des cellules facilitant la comparaison et la mise en avant des informations.
```
Just to avoid having multiple occurences of 'informations' in the same sentence.

MewenLeHo commented on src/fr/articles/tableau-accessible.md:
```diff
@@ -193,15 +204,15 @@ Le premier exemple est un tableau avec seulement des en-têtes sur les colonnes,
-Ce tableau nécessite deux en-têtes, une pour les jours de la semaine et une autre pour le créneau horaire.
+Ce tableau nécessite deux en-têtes, un en-tête de colonne pour les jours de la semaine et un en-tête de ligne pour le créneau horaire.
 
 <table class="table">
```
There is a problem with this example: a `<tr>` tag is missing on line 251 (closed on line 257 but never open).

pya35 committed:
Apply suggestion from @MewenLeHo

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Apply suggestion from @MewenLeHo

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Apply suggestion from @MewenLeHo

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Update accessible-table.md

pya35 committed:
fix missing tag

pya35 committed:
Apply suggestions from code review

Co-authored-by: Mewen Le Hô <92363071+MewenLeHo@users.noreply.github.com>

pya35 committed:
Merge branch 'master' into update-article-tableaux

pya35 commented:
@Aniort ready to be reviewed

pya35 committed:
Merge branch 'master' into update-article-tableaux

pya35 committed:
Merge branch 'master' into update-article-tableaux

pya35 committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo committed:
Merge branch 'master' into update-article-tableaux

MewenLeHo reviewed (approved):
OK
